scripts/support/esp32.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+
+cmd_flash() {
+	local esptool=${ESP_IDF_PATH}/components/esptool_py/esptool/esptool.py
+	local elf_name=${O}/${KERNEL_ELF_NAME}
+
+	echo "Converting ELF to BIN"
+	${esptool} --chip esp32 elf2image ${elf_name}
+
+	echo "Flashing"
+	${esptool} --chip esp32 \
+		--port /dev/ttyUSB0 \
```

Comment:
agree, my USB0 is always occupied by something else, I had to change that when testing ESP32.